### ISSUE TITLE
No notice about serious CVE

### ISSUE THREAD

**Issue by tomchiverton** [flatpak/flatpak.github.io#683]
https://flatpak.org/blog-posts/ doesn't mention recent 10.0 CVE https://nvd.nist.gov/vuln/detail/CVE-2024-42472 ( see https://github.com/flatpak/flatpak/security/advisories/GHSA-7hgv-f2j8-xw87 )

Neither has a press release been added to https://flatpak.org/press/

**Comment by razzeee** [comment]
I don't think that would fit very well into `blog posts` or `press` - it also seems to only have gotten a cve from github and doesn't seem checked by other databases, which makes me wonder about the level

**Comment by tomchiverton** [comment]
So what is the project's policy on publishing vulnerabilities so users / downstream can make informed choices ? Where do they get collected and published ? *Do* they get collected and published ?

**Comment by razzeee** [comment]
I think https://github.com/flatpak/flatpak/blob/main/SECURITY.md#Security-Announcements is
